src/containers/FileDownload.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import { searchStyle } from './theme';
+import { connect } from 'react-redux';
+import Download from '@material-ui/icons/CloudDownload';
+
+class FileDownload extends React.Component {
+  constructor(props) {
+    super(props);
+    this.classes = props.classes;
+  }
+
+  exportCsv(){
```

Comment:
```suggestion
  exportCsv = () => {
```

Reply:
This change might have an issue. The csv file will be automatically downloaded when the user inputs any course. I think there should be a way to avoid this issue.